main.py

```diff
@@ -113,10 +120,25 @@ async def get_compiled_artifact(task_id: str) -> Dict:
     return results[task_id].dict()
 
 
-async def compile_project(project_root: Path, files: List[UploadFile]):
+async def compile_project(project_root: Path, config_override: dict):
     """
     Compile the contrct and asssign the taskid to it
     """
+    (project_root/"ape-config.yaml").write_text(yaml.safe_dump(config_override))
```

Comment:
**question**: do we know for sure this file does not exist? Else you ll be writing to it again and again and it will grow large.

if it is already there, just do this first:

```python
(project_root/"ape-config.yaml").unlink(missing_ok=True)
```

Reply:
good catch, I add this line before it. THank you